proxy/common/_compat.py

```diff
@@ -1,9 +1,19 @@
-"""Compatibility code for using Proxy.py across various versions of Python.
+# -*- coding: utf-8 -*-
```

Comment:
This is only necessary if you support Python 2.
```suggestion
```

Reply:
Totally, I am keeping it for consistency right now.  This is being addressed as part of doc-string fix.